database/models/measurements.js

```diff
@@ -10,8 +10,12 @@ module.exports = {
     create: function (data) {
         return databaseP.then(function (db) {
 
+        var cloneData = JSON.parse(JSON.stringify(data)); // Copy the object...
```

Comment:
@4rzael used this to copy an object, which works in fact. 
@DavidBruant is this ok for you ?